ui/src/__tests__/components/SearchPage.test.tsx

```diff
@@ -110,6 +93,51 @@ describe('Basic search page', () => {
     });
 });
 
+describe('Search page with search type select', () => {
+    const handleSearchFormSubmit = jest.fn((e) => {
+        e.preventDefault();
+    });
+    const setLimit = jest.fn();
+
+    beforeEach(() => {
+        render(
+            <Components.SearchPage
+                handleSearchFormSubmit={handleSearchFormSubmit}
+                isValidating={false}
+                limit={10}
+                offset={0}
+                query={null}
+                results={[]}
+                searchType="publication-versions"
+                setLimit={setLimit}
+                setOffset={jest.fn}
+                showSearchTypeSwitch={true}
```

Comment:
The tests had to change because the search type control is now opt-in with this prop.